### ISSUE TITLE
FhirOperator in evaluateMeasure loads libs twice internally (SDK) into hasmap.

### ISSUE THREAD

**Issue by ahsanbhatti49** [google/android-fhir#1816]
**Describe the bug**
The fhirOperator in evaluateMeasure loads libs twice internally (SDK) into hasmap. It means that if we have 20 measures it would load (fhir-helper, common, +20 measure-libs) * 2. Library size is 3-6MB would always stay into memory if called even once. Implement an unload/clear function in fhirOperator as we are generating measure once and then just load Report from DB.
**Component**
FhirOperator
**To Reproduce**
1. Go to loadLib funtion in fhirOperator.kt '
2. See 
`if (lib.url != null) {
      fhirEngineDal.libs[lib.url] = lib
    }
    if (lib.name != null) {
      libraryContentProvider.libs[lib.name] = lib
    }`

**Expected behavior**
There should be a function to unload these loaded libraries  so that we can free some memory.

**Comment by santosh-pingle** [comment]
@vitorpamplona

**Comment by vitorpamplona** [comment]
Not really. It just doubles the pointer to the library. It doesn't duplicate the object. 

Or am I missing something?

**Comment by ahsanbhatti49** [comment]
There a loop in which We have to generate report for x number of modules, each module contains y indicators, 
for each indicator we are loading library into two hashmaps by name and by url. these libraries will be in the memory until all the measure computed.  If it does not duplicate the object, still it will be there in the memory until all the measures evaluated or may be even after that.

We need a function to remove the loaded library from memory after evaluate measure for each indicator? Otherwise, these libraries will remain in the memory through out the measures generation or may be even after that.

**Comment by jingtang10** [comment]
@ahsanbhatti49 is this still an issue with the knowledge manager library?
